variable/technology/technologies.yaml

```
@@ -460,3 +522,570 @@ Investment|Energy Supply|Liquids|Oil:
                 capturing equipment should be included but not the one on
                 transport and storage.
    unit: billion US$2010/yr
+
+# description of thermal power plants 
+
+Number Units|Electricity|Biomass|Traditionnal:
```

I'm wondering whether `Number of Units` would be more readable...

fixing a typo
```suggestion
Number Units|Electricity|Biomass|Traditional:
```

---

agree